web-app/src/pages/Dashboard/container.js

```diff
@@ -2,30 +2,32 @@ import React from 'react';
 import { PropTypes as PT } from 'prop-types';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
-import { fetchEvents, fetchEventsByUserId } from '../../actions/dashboard';
+import { fetchEvents } from '../../actions/dashboard';
 import { getUser, getTakenHolidays, eventBeingUpdated } from '../../reducers';
+import moment from 'moment';
 
 const DashboardContainer = Wrapped =>
   class extends React.Component {
     static propTypes = {
       userDetails: PT.object,
       fetchEvents: PT.func.isRequired,
-      fetchEventsByUserId: PT.func.isRequired,
       takenEvents: PT.array,
       isEventBeingUpdated: PT.bool,
     };
 
     constructor(props) {
       super(props);
       this.state = {
+        calendarDate: new moment(),
```

Comment:
Could we not format the date here? ie, new moment.format('YYYY-MM-DD')

Reply:
It's best left like this, I think. Keep in mind the next PR will be changing this value by calling other moment methods on this moment instance.